lib/controllers/patients/report.js

```diff
@@ -383,7 +385,7 @@ var setupSchedule = function (data, doc) {
 };
 
 // View PDF report of the patient's data
-report.get("/", auth.authenticate, auth.authorize("read"), function (req, res, next) {
+report.get("/", auth.authenticate, guard(["admin", "programAdministator", "clinician", "user"]), auth.authorize("read"), function (req, res, next) {
```

Comment:
Same question as dump.js

Reply:
Also good catch! I should probably remove "user" from this too.

Reply:
same as dump